src/main/kotlin/blackjack/domain/NumberCard.kt

```diff
@@ -0,0 +1,21 @@
+package blackjack.domain
+
+data class NumberCard(override val symbol: SymbolType, private val number: Int) : Card {
+    override val priority: Int = 1
+
+    init {
+        require(number in numberRange) {
+            "숫자 카드의 값은 $MIN_LIMIT ~ $MAX_LIMIT 사이어야 합니다."
+        }
+    }
+
+    override fun name(): String = number.toString()
+
+    override fun score(acc: Int): Int = number
+
+    companion object {
+        const val MIN_LIMIT = 2
+        const val MAX_LIMIT = 9
+        private val numberRange = (MIN_LIMIT..MAX_LIMIT).toList()
```

Comment:
numberRange의 사용처를 고려했을 때, 꼭 List<Int>로 변환할 필요없이 `IntRange` 타입으로 두셔도 무방할 것 같아요. 😃
```suggestion
        private val numberRange = MIN_LIMIT..MAX_LIMIT
```

이정도면 굳이 상수로 뽑을 필요 없이, 아래와 같이 사용할 수도 있을 것 같아요. 👀

```kotlin
        require(number in MIN_LIMIT..MAX_LIMIT) {
            "숫자 카드의 값은 $MIN_LIMIT ~ $MAX_LIMIT 사이어야 합니다."
        }
```